GuildedRoseLLCSpec/ItemViewControllerSpec.swift

```diff
@@ -42,13 +43,44 @@ class ItemViewControllerSpec: QuickSpec {
             
             describe("viewWillAppear") {
                 it("sets the data source to a static item list") {
+                    
+                    let itemRepository = FakeItemRepository()
+                    itemRepository.stub(items: Item.testData)
+                    controller.itemRepository = itemRepository
+                    
                     controller.viewDidLoad()
                     controller.viewWillAppear(true)
-                    
+
                     let dataSource = controller.itemCollectionView.dataSource as? ItemCollectionViewDataSource
                     
                     expect(dataSource?.items).to(equal(Item.testData))
                 }
+                
+                it("hides the item collection view when there are no items") {
+                    
+                    let itemRepository = FakeItemRepository()
+                    itemRepository.stub(items: [])
+                    controller.itemRepository = itemRepository
+                    
+                    controller.viewDidLoad()
+                    controller.viewWillAppear(true)
+                    
+                    expect(controller.itemCollectionView.isHidden).to(beTrue())
+                    expect(controller.noItemsLabel.isHidden).to(beFalse())
+                }
+                
+                it("hides the item collection view when there are no items") {
+                    
+                    let itemRepository = FakeItemRepository()
+                    itemRepository.stub(items: [Item(name: "FooBar")])
+                    controller.itemRepository = itemRepository
+                    
+                    controller.viewDidLoad()
+                    controller.viewWillAppear(true)
+                    
+                    expect(controller.itemCollectionView.isHidden).to(beFalse())
```

Comment:
I would split this into two tests. Once for each assertion.

Reply:
implemented